test/extended/cli/mustgather.go

```diff
@@ -200,7 +201,17 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 						return nil
 					}
 
-					if (strings.Contains(path, "-termination-") && strings.HasSuffix(path, ".log.gz")) || strings.HasSuffix(path, "termination.log.gz") {
+					if path == "lock.log" {
+						apiserver := filepath.Base(auditDirectory)
+						lockLog, err := ioutil.ReadFile(path)
+						o.Expect(err).NotTo(o.HaveOccurred())
+						result.Flakef("kubelet launched %s without waiting for the old process to terminate (lock was still hold): \n\n%s", apiserver, string(lockLog))
```

Comment:
you don't want to have a separate test for that?
This test is testing `must-gather` not kas's termination

Reply:
done